src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,41 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortButton
+  {:kill-submission
+   (fn [{:keys [props state this]}]
+     (swap! state assoc :stopping-submission? true)
+     (let [submission-id (get (:submission props) "submissionId" )
+           workspace-id (:workspace-id props)
+           path (paths/get-abort-submission-path workspace-id submission-id)
+           headers {"Content-Type" "application/json"}
+           canned-response {:responseText ""
+                            :status 200 :delay-ms (rand-int 2000)}
+           on-done (fn [{:keys [success? status-text]}]
+                     (swap! state assoc :stopping-submission? false)
+                     (if success?
+                       ((:on-kill props))
+                       (js/alert (str "Error in stopping the job : " status-text))))]
+       (utils/ajax-orch path
+         {:headers headers
+          :method "DELETE"
+          :canned-response canned-response
+          :on-done on-done})))
+   :render (fn [{:keys [props state this]}]
+             (if (:stopping-submission? @state)
+               [comps/Blocker {:banner "Stopping submission ..."}]
+               [:div {:style {:padding "0.7em 0" :marginTop "0.5em" :color "#fff" :cursor "pointer"
+                              :backgroundColor (:exception-red style/colors) :borderRadius 5}
+                              :onClick (fn [e]
+                                         (react/call :kill-submission this))}
```

Comment:
Save a line (and some characters, and an unused `e`) by using the `#` function macro.  Also, note that you're not using `this` in `:kill-submission` or `props` in `:render`.